src/unitxt/loaders.py

```diff
@@ -394,3 +395,27 @@ def process(self):
             )
 
         return MultiStream.from_iterables(dataset)
+
+
+class LoadHFCustomDatasetScript(Loader):
+    """Loads a dataset using a custom HF dataset loading script from catalog/loaders path."""
```

Comment:
This need a more extensive documentation maybe with a link to huggingface docs explaining how to create custom datasets as a python file.

Reply:
added further documentation and a link